lib/WP_Auth0_DBManager.php

```diff
@@ -259,6 +274,24 @@ public function notice_failed_client_grant() {
 		}
 	}
 
+	public function notice_successful_client_grant() {
+
+		if ( ! get_option( 'wp_auth0_client_grant_success' ) ) {
+			return;
+		}
+		?>
+		<div class="notice notice-success">
+			<p><?php
+				_e( 'As a part of the 3.4.1 upgrade, a client grant was created for the Auth0 Management API.', 'wp-auth0' );
```

Comment:
I think this should be 3.5.0 now, there are a lot of fixes in this release so it's significant to warrant, same for all 3.4.1 references.